memory/src/main/scala/filodb.memory/data/Shutdown.scala

```diff
@@ -10,6 +12,8 @@ object Shutdown extends StrictLogging {
     forcedShutdowns.increment()
     if (unitTest) throw e
     logger.error(s"Shutting down process since it may be in an unstable/corrupt state", e)
+    import scala.concurrent.duration._
+    Await.result(Kamon.stopModules(), 5.minutes)
```

Comment:
Can we reduce this 1 or 2 minutes please. Do we really want to keep blocking the bad node for 5mins?

Reply:
Almost always this will be done quickly. We are waiting for a maximum of 5 minutes. It would be necessary to have metrics for reliable alerting